libr/core/project.c

```diff
@@ -366,6 +366,32 @@ R_API RThread *r_core_project_load_bg(RCore *core, const char *prjName, const ch
 
 /*** ^^^ thready ***/
 
+
+static ut64 getProjectLaddr(RCore *core, const char *prjfile) {
+	ut64 laddr = 0;
+	FILE *fd;
+	char buf[256];
+	fd = r_sandbox_fopen (prjfile, "r");
```

Comment:
define and assign in the same line

Reply:
what about r_file_slurp + strstr ("\n\"e bin.laddr =")? will be faster and shorter imho

Reply:
copied from r_core_project_info, for good need preload some vars with single pass